sdcm/nemesis.py

```diff
@@ -2616,10 +2616,25 @@ def _mgmt_backup(self, backup_specific_tables):
 
     @latency_calculator_decorator(legend="Scylla-Manger repair")
     def disrupt_mgmt_repair_cli(self):
+        if not self.cluster.params.get('use_mgmt') and not self.cluster.params.get('use_cloud_manager'):
+            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+        if not self.cluster.scylla_manager_tool.feature("repair", "single-host-parallelism"):
+            self.log.info('single-host-parallelism is not supported running normal repair')
+            self.disrupt_mgmt_repair_cli_normal()
```

Comment:
where is this function's definition?

Reply:
shouldn't it be just the same as the below `normal` mgmt repair? AFAIU it needs still all the definition of asserting if the task is not done.... that now is missing from this call

Reply:
removed this part based on @roydahan comment